x-pack/platform/plugins/private/transform/public/alerting/transform_alerting_flyout.tsx

```diff
@@ -32,44 +32,42 @@ export const TransformAlertFlyout: FC<TransformAlertFlyoutProps> = ({
   onCloseFlyout,
   onSave,
 }) => {
-  const { triggersActionsUi } = useAppDependencies();
+  const { triggersActionsUi, ...plugins } = useAppDependencies();
 
   const AlertFlyout = useMemo(() => {
     if (!triggersActionsUi) return;
 
     const commonProps = {
-      onClose: () => {
+      plugins,
```

Comment:
ML and Transform code changes LGTM, but I noticed some issues during local testing. I’ve left comments on the PR separately.